researcher_ids/RI_Claims_V1.md

```diff
@@ -536,9 +533,9 @@ the GA4GH DURI committee.
 
 #### "**value**"
 
--   REQUIRED. A string that represents the type, process and version of the
-    claim. The format of the string can vary by the [Passport Visa
-    Type](#passport-visa-type).
+-   REQUIRED. A string that represents any of the scope, process, identifier and
+    version of the assertion. The format of the string can vary by the [Passport
+    Visa Type](#passport-visa-type).
 
 -   For example, `value` =
     "<https://doi.org/10.1038/s41431-018-0219-y>" when `type` =
```

Comment:
Perhaps enhance the wording to give the idea that there may be also other definitions of registered access.

Reply:
Done